src/navigator/main-stack.tsx

```diff
@@ -82,7 +86,11 @@ export const MainStackScreen = () => {
     dispatch(loadExchangeRates.submit());
   };
 
-  useAuthorisedTimerEffect(initDataLoading, DATA_REFRESH_INTERVAL, [selectedAccount.publicKeyHash, selectedRpcUrl]);
+  useTimerEffect(initDataLoading, DATA_REFRESH_INTERVAL, [
```

Comment:
why did you change `useAuthorisedTimerEffect` to `useTimerEffect`?

Reply:
because there was simple settimeout change at beginning, but it was not firing events more than once